test/constants.ts

```diff
@@ -107,3 +107,18 @@ export const sampleRateModel = {
   R1: toWei(0.07).toString(),
   R2: toWei(0.75).toString(),
 };
+
+export const cctpTokenMessengerAbi = [
+  {
+    inputs: [
+      { internalType: "uint256", name: "amount", type: "uint256" },
+      { internalType: "uint32", name: "destinationDomain", type: "uint32" },
+      { internalType: "bytes32", name: "mintRecipient", type: "bytes32" },
+      { internalType: "address", name: "burnToken", type: "address" },
+    ],
+    name: "depositForBurn",
+    outputs: [{ internalType: "uint64", name: "_nonce", type: "uint64" }],
+    stateMutability: "nonpayable",
+    type: "function",
+  },
+];
```

Comment:
I don't think this is needed. See: https://github.com/across-protocol/contracts-v2/blob/51f7a0474b4d493911e1a81f864eb5f86139f444/utils/abis.ts

Reply:
ahh wasn't aware that we have this already. will remove this redundant one